test/uk/gov/hmrc/plasticpackagingtax/registration/controllers/group/OrganisationDetailsTypeControllerSpec.scala

```diff
@@ -151,7 +154,15 @@ class OrganisationDetailsTypeControllerSpec extends ControllerSpec {
           val result      = controller.submitNewMember()(postJsonRequestEncoded(correctForm: _*))
           mockCreatePartnershipGrsJourneyCreation("http://test/redirect/partnership")
           redirectLocation(result) mustBe Some("http://test/redirect/partnership")
+
+          val registrationCaptor: ArgumentCaptor[Registration] =
+            ArgumentCaptor.forClass(classOf[Registration])
+          verify(mockRegistrationConnector, times(2)).update(registrationCaptor.capture())(any())
```

Comment:
Performing 2 discrete updates to the BE feels a little wrong. I guess this might go away if we move the registration updating logic back into the group `OrganisationDetailsTypeControllerBase`?